pina/utils.py

```diff
@@ -41,5 +42,20 @@ def merge_two_tensors(tensor1, tensor2):
 
     tensor1 = LabelTensor(tensor1.repeat(n2, 1), labels=tensor1.labels)
     tensor2 = LabelTensor(tensor2.repeat_interleave(n1, dim=0),
-                            labels=tensor2.labels)
+                          labels=tensor2.labels)
     return tensor1.append(tensor2)
+
+
+class MyDataSet(torch.utils.data.Dataset):
```

Comment:
Try to avoid `MyClass`, `MyData`, etc.
What about `SamplesDataSet`?

Reply:
solved in commit [b9cb585](https://github.com/mathLab/PINA/pull/51/commits/b9cb585e0ab2ca4598a5e05ae7c2241ce3e4beb1)